app/views/messages/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <% user = User.find(message.user_id) %>
       <article class="messages">
         <div class="message-body">
-          <div class="inline-block"><strong><%= user.username %></strong>
+          <div class="inline-block"><strong><%= user.username %></strong> <%= message.sent_time %>
```

Comment:
I was wonder if you are keeping username twice in one message on purpose? I think it would be better is we say “Message sender at: “ on line 11.